docs/section19/19-14.md

```diff
@@ -0,0 +1,71 @@
+# Section 19 - Rule 19-14             
+**Schema Version:** 0.0.23    
+**Mandatory Rule:** True    
+**Rule ID:** 19-14               
+**Rule Description:**  For baseline system types 1-8 and 11-13, if return or relief fans are specified in the proposed design, the baseline building design shall also be modeled with fans serving the same functions and sized for the baseline system supply fan air quantity less the minimum outdoor air, or 90% of the supply fan air quantity, whichever is larger.
+
```

Comment:
I don't think eQUEST has "relief" fan type. Does E+? Can releif fan be modeled as exhaust fan?

Reply:
eQuest has options for all 3. Discuss on call.
![image](https://user-images.githubusercontent.com/92933945/217647620-652aeb06-c013-451a-a66d-61a455b7bde2.png)

Reply:
See below, waiting.